lib/tests/select.rs

```diff
@@ -107,3 +107,84 @@ async fn select_writeable_subqueries() -> Result<(), Error> {
 	//
 	Ok(())
 }
+
+#[tokio::test]
+async fn select_where_field_is_bool() -> Result<(), Error> {
+	let sql = "
+		CREATE test:1 SET active = false;
+		CREATE test:2 SET active = false;
+		CREATE test:3 SET active = true;
+		SELECT * FROM test WHERE active = false;
+		SELECT * FROM test WHERE active != true;
+	";
```

Comment:
The code looks correct to me so I think this will obviously pass but I think it will be nice to also add

```sql
SELECT * FROM test WHERE active = true;
```

to make sure we cover our bases to future-proof changes to this part or the codebase.

Reply:
Done!

Reply:
Thanks!